resources/qml/delegates/ImageMessage.qml

```diff
@@ -68,7 +56,63 @@ AbstractButton {
         eventId: parent.eventId
     }
 
-    onClicked :Settings.openImageExternal ? room.openMedia(eventId) : TimelineManager.openImageOverlay(room, url, eventId, originalWidth, proportionalHeight);
+    Image {
+        id: blurhash_
+
+        anchors.fill: parent
+        visible: img.status != Image.Ready || timeline.privacyScreen.active
+        source: blurhash ? ("image://blurhash/" + blurhash) : ("image://colorimage/:/icons/icons/ui/image-failed.svg?" + Nheko.colors.buttonText)
```

Comment:
if we have no blurhash, this doesn't work well

Reply:
I realized that. I plan to add a local blurhash generator to handle that case.

Reply:
Or you just make the image fade out instead of the blurhash. Then it will just fade to the background if there is no blurhash.

Reply:
That doesn't work as well when you have an image with transparent parts, because the blurhash then shows behind it.

Reply:
I guess that makes sense. Although you could cross-fade them.

Reply:
It turns out I'm too lazy to actually do the blurhash generator thingy, so I just made it crossfade to the background if necessary :P